storybook/storybook-css/src/components/Heading/story-template.jsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * @license EUPL-1.2
+ * Copyright (c) 2022 Gemeente Utrecht
+ * Copyright (c) 2022 Frameless B.V.
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import React from 'react'
+
+export const argTypes = {
+  textContent: {
+    table: {
+      type: { summary: 'string' },
+    },
+  },
+  level: {
+    options: [1, 2, 3, 4, 5, 6],
+    control: 'select',
+    table: {
+      type: { summary: 'number' },
+      defaultValue: { summary: '1' },
+    },
+  },
```

Comment:
I think we can remove this control for the css component. It only renders a different html element, which is something you should handle outside of the component library I think.

Reply:
If you don't define these for the CSS controls will show as 'unknown' and only have a text input for value. Which would allow you to create a `H32` tag or something.

Reply:
The react variant will index the type of value, which is cooler obviously.